### PR TITLE
Updated graphics readme

### PR DESCRIPTION
**Pull request by phi-lira**
### Purpose of this PR
 Updated graphics README with latest information.

---
### Testing status
 Tested new links to see if they were working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,32 +27,27 @@ com.unity.render-pipelines.high-definition-config | [![](https://badge-proxy.cds
...
 com.unity.visualeffectgraph | [![](https://badge-proxy.cds.internal.unity3d.com/0fbfa6fc-2faf-4689-a3e7-fca736ab23cb)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/build-info?branch=master) [![](https://badge-proxy.cds.internal.unity3d.com/6606630d-31a9-4af5-b63c-25272411c381)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/dependencies-info?branch=master) [![](https://badge-proxy.cds.internal.unity3d.com/c10f50c2-2a79-4d0a-a763-54dcb40d027f)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/dependants-info) [![](https://badge-proxy.cds.internal.unity3d.com/823df233-071e-4ceb-a39f-b810d7fe6fe1)](https://badges.cds.internal.unity3d.com/packages/com.unity.visualeffectgraph/warnings-info?branch=master) ![ReleaseBadge](https://badge-proxy.cds.internal.unity3d.com/59b6ec9b-c477-4767-82ba-d2390e70cede) ![ReleaseBadge](https://badge-proxy.cds.internal.unity3d.com/ae2fb4f5-43dc-4ad2-8c94-7190dbcdc132)
 
-## Using the latest version
```
Comment (phi-lira) [inline README.md:30]: This `using the latest version` section were outdated and described workflows related to package versioning which don't make sense in newer unity versions anymore. I moved the section at the bottom of the document and renamed it to  `## Package versions on Unity 2020.3 LTS and below`

**Review by jessebarker — APPROVED** [review]
LGTM

**Review by theo-pnv — COMMENTED** [review]
Nice, thanks for taking care of this!

Is the `Package CI Summary` section at the top of the readme still relevant? I find it a little bit confusing for external users, as we are only showing the status of 10.x.x without providing any more context (which unity release it's linked to for example). I'm also not sure what most badges actually refer to ("dependant"? "warnings"?..). cc @sophiaaar , I vaguely remember you participated in setting up these badges?

Finally, maybe we should add some links for the postprocessing package, now that it was moved back here. At the top of the readme there's the list of all packages with links to the docs, we should probably add that one https://docs.unity3d.com/Manual/PostProcessingOverview.html (cc @sebastienlagarde)

**Comment by theo-pnv** [comment]
On a sidenote I'm seeing the formatting job failing so don't hesitate to use the new "push required format changes from yamato" job here if needed :) https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/update-repository-readme/.yamato%252F_formatting.yml%2523formatting_apply

**Pull request merged**